extensions/cache/deployment/src/main/java/io/quarkus/cache/deployment/CacheProcessor.java

```diff
@@ -113,4 +114,13 @@ void recordCachesBuild(CombinedIndexBuildItem combinedIndex, BeanContainerBuildI
         }
         return cacheNames;
     }
+
+    private static class IsEnabled implements BooleanSupplier {
```

Comment:
Minor nitpick: Can we call give this a more descriptive name, something like `CacheEnabled` or something?
That would make it more consistent with the the existing uses of `onlyIf`

Reply:
That's a good point, I just applied your suggestion. Thanks for reviewing this!